src/components/EventCard.astro

```diff
@@ -0,0 +1,106 @@
+---
+import { classNames } from "../utils/classNames";
+
+type Props = {
+  className?: string;
+  title: string;
+  img: string;
+  date: string;
+  link: string;
+  hoverTextColor?: "black" | "white";
+};
+
+const {
+  className,
+  title,
+  img,
+  date,
+  link,
+  hoverTextColor = "black",
+} = Astro.props;
+const hoverTextShadowColor =
+  hoverTextColor === "black" ? "#ffffff7f" : "#0000007f";
+---
+
+<div
+  class={classNames(
+    className,
+    "grid grid-cols-8 grid-rows-11 w-1/2 md:w-3/12 aspect-[18/24]"
+  )}
+>
+  <a
+    target="_blank"
+    href={link}
+    class="bg-group peer rounded-lg shadow-2xl bg-wsg-orange-500 col-start-1 col-end-8 row-start-1 row-end-11"
+  >
+  </a>
+  <a
+    target="_blank"

```

Comment:
When using `target="_blank"` you should always set `rel="noopener noreferrer"` (https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever)